cypress/e2e/Profile/Alerts/ProfileCompletionSidebarNotification.feature

```diff
@@ -1,18 +1,27 @@
+@profile @profile-completion @profile-completion-sidebar @business:low-impact
```

Comment:
Low impact because the banner is present in more pages, and we run the test for the banner